modern-business/sass/blocks/_blocks.scss

```diff
@@ -200,7 +200,7 @@
 		}
 
 		@include media(tablet) {
-			padding: 60px 0;
+			padding: $size__vertical-spacing-unit 0;
```

Comment:
This vertical padding inside a media and text block in this theme was 60px before, and it now makes sense to change a multiple of 32px with this change, but 32px feels a bit too small. How about 64px? You can find many 60px used around this block, and it's probably better to change all of them.

Reply:
Got it. I’ll sort them all out so they’re all using the variable with the multiplier.